### ISSUE TITLE
Relationship to IsoCor CRAN package in R

### ISSUE THREAD

**Issue by hechth** [MetaSys-LISBP/IsoCor#39]
The package lister [here](https://cloud.r-project.org/web/packages/IsoCor/index.html) has the same name and seems to be for a similar application - is there any relationship or did you just choose the same name?

**Comment by pierremillard** [comment]
Hello, there is no relationship. We have published the first release of IsoCor in 2012 ([here](https://academic.oup.com/bioinformatics/article/28/9/1294/312644)) and the second release in 2019 ([here](https://academic.oup.com/bioinformatics/article/35/21/4484/5418798)), so I guess they just choose the same name...

**Comment by hechth** [comment]
Thanks a lot for making this clear :) Maybe something could be included in the Readme, just to make sure that people are aware?

**Comment by pierremillard** [comment]
yes, we'll include a notice in the doc & readme, thanks for this information! :)
